### ISSUE TITLE
Chrome: Use current domain as host

### ISSUE THREAD

**Issue by jgallen23** [livereload/livereload-extensions#15]
I'm developing on a dev server so the host is not http://localhost.  It would be nice to have the chrome extension use the current host instead of defaulting to 127.0.0.1.

**Comment by xhoskin** [comment]
+1

**Comment by camilonova** [comment]
Please, anyone found a workaround?

**Comment by camilonova** [comment]
If you include the script manually will work:

```
<script>document.write('<script src="//' + location.host.split(':')[0] + ':35729/livereload.js"></' + 'script>')</script>
```

But this should be fixed ASAP.

**Comment by andreyvit** [comment]
Guys, a change to try to connect to _both_ 127.0.0.1 and the tab's host is planned. I cannot make it connect to _only_ the tab's host because that will break valid remote use cases of my app. (The app can override remote CSS files with local ones, so users can be editing styles for a site running remotely.)

**Comment by camilonova** [comment]
@andreyvit how about merging #35 ?

**Comment by andreyvit** [comment]
@camilonova But I've just explained why #35 cannot be merged. :-)

The only way to solve this is to try to connect to both hosts. I'll get to that soon. If someone wants to do a pull request sooner, I'll be happy to merge it.

**Comment by camilonova** [comment]
@andreyvit Thank you, do you have any updates about this?

**Comment by andreyvit** [comment]
@camilonova Nope, I didn't mean _that_ soon. Contributions are still welcome. :-)

**Comment by camilonova** [comment]
@andreyvit If you give me some directions I can try to do it on my own

**Comment by cj** [comment]
+1

**Comment by camilonova** [comment]
@andreyvit any updates?

**Comment by ChristopherDosin** [comment]
+10000

**Comment by camilonova** [comment]
Do we have any updates, or there is way I can help?

**Comment by lokcito** [comment]
I have just upload a custom version that allow set your custom host and port: 
Here the link: https://github.com/lokcito/livereload-extensions 
Download here: https://github.com/lokcito/livereload-extensions/raw/master/dist/2.1.0/LiveReload-2.1.0-ChromeWebStore.zip

**Comment by maksymz** [comment]
+1 for having ability to configure host/port (or least host, or at least use current website domain). Required to be able to use extension while developing in Docker.
